frontend/src/components/Login.jsx

```
@@ -56,63 +73,97 @@
-            size='large'
-            type='submit'
-            disabled={isLoading}
-          >
+          {error && <p className="text-red-500 text-center">{error}</p>}
+          <ColorButton variant="contained" size="large" type="submit" disabled={isLoading}>
             {isLoading ? 'Logging in...' : 'Login'}
           </ColorButton>
-          <div className='flex items-center gap-2 text-sm mt-4'>
-            <div className='w-full h-[1px] bg-gray-300'></div>
+          <div className="flex items-center gap-2 text-sm mt-4">
+            <div className="w-full h-[1px] bg-gray-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)